src/tests/banners/bannerSelection.test.ts

```diff
@@ -0,0 +1,90 @@
+import { readerRevenueRegionFromCountryCode } from './bannerSelection';
+import { SubscriptionsBanner } from './SubscriptionsBannerTest';
+import { WeeklyBanner } from './WeeklyBannerTest';
+
+describe('readerRevenueRegionFromCountryCode', () => {
+    it('should return a region', () => {
+        const unitedKingdom = readerRevenueRegionFromCountryCode('GB');
+        expect(unitedKingdom).toBe('united-kingdom');
+
+        const germany = readerRevenueRegionFromCountryCode('DE');
+        expect(germany).toBe('european-union');
+
+        const australia = readerRevenueRegionFromCountryCode('AU');
+        expect(australia).toBe('australia');
+
+        const fiji = readerRevenueRegionFromCountryCode('FJ');
+        expect(fiji).toBe('rest-of-world');
+
+        const usa = readerRevenueRegionFromCountryCode('US');
+        expect(usa).toBe('united-states');
+    });
+});
+
+describe('SubscriptionsBanner canRun', () => {
```

Comment:
There are a few other things we could test here like the `alreadyVisitedCount` or the `shouldHideReaderRevenue` flag

Reply:
Definitely, thanks for the feedback